tests/decomp/030-fun_bfffd0eb/fun_bfffd0eb.lst.exp.clean

```diff
@@ -3,12 +3,11 @@ void fun_bfffd0eb()
 {
   if ($a2_0 == 0x3) {
     $a2 = 0xb;
-  } else if ($a2_0 != 0x6) {
-    $a2 = 0x0;
-    if ($a2_0 == 0xc) {
-      $a2 = 0xd;
-    }
-  } else {
+  } else if ($a2_0 == 0x6) {
     $a2 = 0xc;
+  } else if ($a2_0 == 0xc) {
```

Comment:
So, this is of course the reason why the new pass is added: with it, this example, as intended, turns into if-else chain, equivalent to a switch statement.